image/templates/helm/stackrox-central/Chart.yaml.htpl

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2 # Can probably be generalized to v1 later. TODO(ROX-5502).
 name: stackrox-central-services
-icon: https://www.stackrox.com/img/logo.svg
+icon: https://github.com/stackrox/stackrox/blob/master/image/templates/helm/stackrox-central/assets/icon.png
```

Comment:
Hm. It is a bit concerning that this is the logo for both Community and Commercial charts. How hard would it be to switch logo per active branding?